src/otx/core/model/entity/base.py

```diff
@@ -248,9 +249,14 @@ def _export_parameters(self) -> dict[str, Any]:
             ("model_info", "label_ids"): all_label_ids.strip(),
             ("model_info", "optimization_config"): json.dumps(optimization_config),
         }
+        parameters["custom_forward"] = self.get_custom_forward()
 
         return parameters
 
+    def get_custom_forward(self):
```

Comment:
I'd keep it as an private property. Can we consider renaming this to `_explain_forward` for clarity?

Reply:
Fixed and renamed